home/views.py

```diff
@@ -59,6 +77,26 @@ def privacy_policy(request):
     return render(request, "privacy-policy.html")
 
 
+def save_partnership_contact_form(request):
+    """ Saves the requests coming from the Partnership Contact form on the
+    home page """
+    if request.method == 'POST':
+        form = PartnershipRequestForm(request.POST)
+        if form.is_valid():
+            form.save()
+            send_partnership_request_email(request.POST)
+            messages.success(request, "Thank you very much for your interest! We will be in touch shortly.")  # noqa: E501
```

Comment:
This should be able to be broken into two lines, why are we forcing `#noqa` here?

Reply:
I have broken this into two lines.